challenges/objects-arrays.js

```diff
@@ -5,13 +5,44 @@
   Use this pattern to create your objects: 
   object name, diet, weight, length, period
 */
-
+function Dinosaur(attr) {
```

Comment:
This is a valid way of creating all of the dinosaurs. In the constructor function, though, there isn't a constructor method.